### DISCUSSION TITLE
wstunnel v3.1  suport Openwrt arm64

### DISCUSSION THREAD

**Discussion by sihiver** [erebe/wstunnel#119] [Q&A]
why wstunnel v4.1 not suport openwrt ?
i build wstuunel v3.1 from source not working on openwrt, how to build wstunnelv3.1 ?

**Comment by erebe** [comment]
Hello,

Only because I don't have the OS/device I don't provide the binary, wstunnel should be able to compile on it.
So feel free to give it a try yourself

**Comment by esaaprillia** [comment]
should be able to run on openwrt

**Comment by Milhouse77** [comment]
It indeed works on Openwrt.
I compiled it for two targets (Netgear R7800 (armv7-unknown-linux-musleabihf) and Newifi D2 (mipsel-unknown-linux-musl).
